class/streamtop3.var

```diff
@@ -1,6 +0,0 @@
-STREAM_RTSP_IP=192.168.48.4:554
```

Comment:
I am okay removing this file -- but maybe leave an example one for later use?
This is for Hypatia at conferences.